kokoro/linux/build.sh

```diff
@@ -101,12 +101,23 @@ $SRC/kokoro/linux/package.sh $BUILD_ROOT/out
 ## Swarming tests, see test/swarming/README.md
 ##
 
-# Install LUCI
-curl -fsSL -o luci-py.tar.gz https://chromium.googlesource.com/infra/luci/luci-py.git/+archive/2128d8d9c36a0e2839afa200cf3da5e6f6ea845a.tar.gz
-mkdir luci-py
-tar xzf luci-py.tar.gz --directory luci-py
-export LUCI_CLIENT_ROOT="$PWD/luci-py/client"
-
+# "Swarming" is a tool from Chromium LUCI, install using the recommended CIPD
+# from depot_tool
+export LUCI_ROOT="`pwd`/luci"
+mkdir -p ${LUCI_ROOT}
+
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+export PATH="`pwd`/depot_tools:$PATH"
+
+# You can get valid git revision by looking up infra/tools/... at:
+# https://chrome-infra-packages.appspot.com/
+# Note that the '${platform}' must appear as-is in the ensure file,
+# hence the single quotes.
+(
+  echo 'infra/tools/luci/isolate/${platform} git_revision:227ed2302d4b77f5a9823c223d5c1571b45dafd4'
+  echo 'infra/tools/luci/swarming/${platform} git_revision:227ed2302d4b77f5a9823c223d5c1571b45dafd4'
```

Comment:
Just double checking you meant to put the same revision here.

Reply:
Yep, it's the same revision.